apps/juicebox_web/assets/js/videos/video-placeholder.scss

```diff
@@ -0,0 +1,17 @@
+.video-placeholder {
+  display: flex;
+  background-color: #f2f2f2;
+}
+
+.video-name {
+  padding: 10px;
+}
+
+.video-voting {
+  display: flex;
+  flex-flow: column;
+  justify-content: space-between;
+  height: 100%;
```

Comment:
Properties should be ordered display, width, height

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)